### ISSUE TITLE
KeyError: 'close'

### ISSUE THREAD

**Issue by kau50** [PhongCode/Stock-prediction-2021#3]
KeyError
KeyError: 'close'

File "D:\stockmarket-prediction-master\app.py", line 25, in requestStock
return predictStock(stockName)
File "D:\stockmarket-prediction-master\app.py", line 13, in predictStock
stockData=stockpredict(stockName)
File "D:\stockmarket-prediction-master\main.py", line 36, in stockpredict
cl = data[data['Name']==stockName]['close']

i am getting this error, how do i fix it?[
![image](https://user-images.githubusercontent.com/72035790/235603337-1e7cf551-c791-435c-9946-039274d4fcee.png)
](url)

**Comment by mrtrieuphong** [comment]
Please check your dataset, ensure that the `close` column is available.

**Comment by kau50** [comment]
yes i verified that close column is available, i have used the dataset provided in the repo. I switched close column from E to F, but that does solve the issue.
![image](https://user-images.githubusercontent.com/72035790/235672139-34815b8c-33b5-4a71-843f-047846e4adb5.png)

**Comment by mrtrieuphong** [comment]
Can you `print(data.head())` and `print(data[stockName])` before `cl` variable?

**Comment by kau50** [comment]
this is the output 
getting a keyerror:AAPL
![image](https://user-images.githubusercontent.com/72035790/235717025-4469a279-2609-4c9d-9168-846906ffd958.png)
![image](https://user-images.githubusercontent.com/72035790/235717083-9f13249b-bb89-4bce-84c0-7fb76aa892b6.png)
![image](https://user-images.githubusercontent.com/72035790/235717315-24dfe36e-4e6c-4027-bfcc-d714a024f829.png)
